backend/src/main/resources/application.yaml

```diff
@@ -35,13 +35,22 @@ spring:
           keycloak:
             authorization-url: ${keycloak.base-oauth-url}/protocol/openid-connect/auth
             token-uri: ${keycloak.base-oauth-url}/protocol/openid-connect/token
+  datasource:
+    url: jdbc:postgresql:postgres
+    username: postgres
+    password: ${POSTGRES_PASSWORD}
+    database : organizations
+    organization_id : organization_id
+    organization_name : organization_name
+    password_encryption : md5
```

Comment:
Is the password actually MD5 encrypted? I was able to enter a password without encrypting it...

Typically we don't encrypt environment variable machine passwords, as we would need to provide a master password to decrypt those passwords and so on. There are some exceptions.

Reply:
This was an attempt to solve some other issue. I'll delete this